src/ohio/pipeio.py

```diff
@@ -223,3 +250,15 @@ def writable(self):
             raise baseio.IOClosed()
 
         return True
+
+
+def pipe_text(writer_func, *args, buffer_size=None, **kwargs):
```

Comment:
The way you introduced it in slack this sounded like it was going to do more, but it appears that you've delegated everything to PipeTextIO. Does this helper function need to exist?

Reply:
On second look allowing the splatted arguments is nice. I'm slightly concerned that throwing in the buffer_size that doesn't apply to the called function but rather to the PipeTextIO might be confusing to people reading code written with `pipe_text`

Reply:
Agreed. But that's the reason to push this to the syntactic helper.